.rubocop.yml

```diff
@@ -157,6 +158,13 @@ Lint/ModuleDisclosureDatePresent:
     # Only exploits require disclosure dates, but they can be present in auxiliary modules etc.
     - 'modules/exploits/**/*'
 
+Lint/DeprecatedGemVersion:
+  Enabled: true
+  Exclude:
+    - 'metasploit-framework.gemspec'
+    - 'spec/**/*'
+    - 'lib/rex/version.rb'
```

Comment:
Thoughts on adding the rubocop ignores to the `version.rb` and `version_spec.rb` files directly, rather than these global ignores? 👀

Reply:
By that I mean placing these in the files
```
# rubocop:disable Lint/DeprecatedGemVersion
```

Reply:
hmmm, I think that would make sense for `lib/rex/version.rb` for sure
I'm not so sure about the specs though...I guess I was kinda thinking I didn't want to restrict what you were allowed to do in the tests? but honestly it doesn't bother me much one way or the other, happy to make the change